src/util/StreamLink.cpp

```diff
@@ -205,6 +209,11 @@ void openStreamlink(const QString &channelURL, const QString &quality,
 
 void openStreamlinkForChannel(const QString &channel)
 {
+    auto *currentPage = dynamic_cast<SplitContainer *>(
+        getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
+    currentPage->getSelectedSplit()->getChannel()->addMessage(
+        makeSystemMessage(QString("Opening %1 in streamlink...").arg(channel)));
```

Comment:
```suggestion
    static const QString INFO_TEMPLATE("Opening %1 in Streamlink ...");

    auto *currentPage = dynamic_cast<SplitContainer *>(
        getApp()->windows->getMainWindow().getNotebook().getSelectedPage());
    currentPage->getSelectedSplit()->getChannel()->addMessage(
        makeSystemMessage(INFO_TEMPLATE.arg(channel)));
```

This prevents a temporary QString from being created on every function call.

Reply:
Oh yea, I didn't think about this... Thank god there's smarter programmers

Reply:
It's really a nitpick in this case, not likely to ever cause performance problems. But while we're on it, we might as well make it the most correct it can be :man_shrugging: